std/process.d

```diff
@@ -525,8 +523,11 @@ private Pid spawnProcessImpl(in char[] commandLine,
                               out int fileDescriptor, out HANDLE handle)
     {
         fileDescriptor = getFD(file);
-        if (fileDescriptor < 0)   handle = GetStdHandle(stdHandle);
-        else                      handle = file.windowsHandle;
+        handle = null;
+        if (fileDescriptor >= 0)
+            handle = file.windowsHandle;
+        if (handle is null || handle == INVALID_HANDLE_VALUE)
+            handle = GetStdHandle(stdHandle);
```

Comment:
This distinction between `null` and `INVALID_HANDLE_VALUE` seems gratuitous.

perhaps:
```D
if(fileDescriptor < 0 || (handle = file.windowsHandle) == INVALID_HANDLE_VALUE)
   handle = GetStdHandle(stdHandle);
```